// jquery-ajax-json/dist/jquery-ajax-json.js
(function(f){if(typeof exports==="object"&&typeof module!=="undefined"){module.exports=f()}else if(typeof define==="function"&&define.amd){define([],f)}else{var g;if(typeof window!=="undefined"){g=window}else if(typeof global!=="undefined"){g=global}else if(typeof self!=="undefined"){g=self}else{g=this}g.jqueryAjaxJson = f()}})(function(){var define,module,exports;return (function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
var $ = window.$ || window.jQuery || require('jquery')
/**
 * Shorthand for posting JSON
 *
 * @param  {[type]}   url      [description]
 * @param  {[type]}   data     [description]
 * @param  {Function} callback [description]
 * @return {[type]}            [description]
 */
module.exports = function (url, data, callback) {
  var settings = {
    url: url,
    contentType: 'application/json',
    type: 'POST',
    data: JSON.stringify(data)
  }
  if (callback) {
    settings.success = callback
  }
  return $.ajax(settings)
}

if ($) {
  $.postJSON = module.exports
}

},{"jquery":"jquery"}]},{},[1])(1)
});
